independent-publisher-connectors/Rarible/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3342 @@
+      "type": "object",
+      "properties": {
+        "date": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "makeValue": {
+          "example": "717802.342336",
+          "type": "number"
+        },
+        "takeValue": {
+          "example": "717802.342336",
+          "type": "number"
+        }
+      },
+      "required": [
+        "date",
+        "makeValue",
+        "takeValue"
+      ]
+    },
+    "OrdersPagination": {
+      "title": "OrdersPagination",
+      "type": "object",
+      "properties": {
+        "orders": {
+          "description": "Orders list",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Order"
+          }
+        },
+        "continuation": {
+          "description": "Continuation token from the previous response",
+          "type": "string"
+        }
+      },
+      "required": [
+        "orders"
+      ]
+    },
+    "OrderCurrencies": {
+      "title": "OrderCurrencies",
+      "description": "List of distinct currencies in which payments for sell or bid orders were suggested",
+      "type": "object",
+      "properties": {
+        "orderType": {
+          "$ref": "#/definitions/OrderType"
+        },
+        "currencies": {
+          "type": "array",
+          "items": {
+            "type": "object"
+          }
+        }
+      },
+      "required": [
+        "orderType",
+        "currencies"
+      ]
+    },
+    "Code1": {
+      "title": "Code1",
+      "description": "Error type",
+      "example": "UNKNOWN",
+      "type": "string",
+      "enum": [
+        "UNKNOWN",
+        "VALIDATION",
+        "TOKEN_NOT_FOUND",
+        "BALANCE_NOT_FOUND"
+      ]
+    },
+    "Code2": {
+      "title": "Code2",
+      "description": "Error type. `BAD_REQUEST` - incorrect format of input data, `VALIDATION` - input data format is correct, but its content doesn't meet requirements of API call",
+      "example": "BAD_REQUEST",
+      "type": "string",
+      "enum": [
+        "BAD_REQUEST",
+        "VALIDATION"
+      ]
+    },
+    "Feature": {
+      "title": "Feature",
+      "example": "APPROVE_FOR_ALL",
+      "type": "string",
+      "enum": [
+        "APPROVE_FOR_ALL",
+        "SET_URI_PREFIX",
+        "BURN",
+        "MINT_WITH_ADDRESS",
+        "SECONDARY_SALE_FEES",
+        "MINT_AND_TRANSFER"
+      ]
+    },
+    "OrderType": {
+      "title": "OrderType",
+      "example": "SELL",
+      "type": "string",
+      "enum": [
+        "SELL",
+        "BID"
+      ]
+    },
+    "Type1": {
+      "title": "Type1",
+      "description": "Collection type",
+      "example": "ERC721",
+      "type": "string",
+      "enum": [
+        "ERC721",
+        "ERC1155",
+        "CRYPTO_PUNKS"
+      ]
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {},
+  "security": [],
+  "tags": [
+    {
+      "name": "erc20-balance-controller",
+      "description": ""
```

Comment:
@RoyPaarMS Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
Removed